### ISSUE TITLE
Calling spacemacs/edebug-instrument-defun-off produces an error.

### ISSUE THREAD

**Issue by soulawaker** [syl20bnr/spacemacs#10068]
#### Description :octocat:
Calling `spacemacs/edebug-instrument-defun-off` produces an error.

#### Reproduction guide :beetle:
- Start Emacs
- Move point to an defun scope in any .el file.
- `,-d-f` (`spacemacs/edebug-instrument-defun-on`)
- Go some steps by `s`.
- `,-d-F` (`spacemacs/edebug-instrument-defun-off`)
- An error occured.

*Observed behaviour:* :eyes: :broken_heart:
An error occured.

*Expected behaviour:* :heart: :smile:
Just off defun instrument.

#### System Info :computer:
- OS: gnu/linux
- Emacs: 24.5.1
- Spacemacs: 0.200.10
- Spacemacs branch: master (rev. 4bb4cb4)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(sql go yaml javascript html helm auto-completion emacs-lisp clojure java php docker latex git github markdown org
     (spell-checking :variables enable-flyspell-auto-completion t spell-checking-enable-by-default nil)
     colors finance blog)
```

#### Backtrace :paw_prints:
In \*Messages\* buffer
```
apply: Wrong number of arguments: #[(ad--addoit-function edebug-it) "Æ	DÇÆÈÉÊ Ë ÌÍ!)!@
A@Î
8Ï
8:ÐÑ
$@!-J!)" [ad-return-value eval-sexp-fu-flash-mode --cl-var-- bounds hi unhi nil eval-sexp-fu-flash (preceding-sexp) ((error)) ...] 6], 1
```

**Comment by CeleritasCelery** [comment]
This is fixed in develop with #8510

**Comment by soulawaker** [comment]
Then, I'll try it with the ​develop branch. Thanks.
